python/lsst/daf/butler/core/configSupport.py

```diff
@@ -153,6 +156,46 @@ def clone(self, name=None, dimensions=None, dataId=None):
         return self.__class__(name=name, dimensions=dimensions, dataId=dataId)
 
 
+def normalizeLookupKeys(toUpdate, universe):
+    """Normalize dimensions used in keys of supplied dict.
+
+    Parameters
+    ----------
+    toUpdate : `dict` with keys of `LookupKey`
+        Dictionary to update.  The values are reassigned to normalized
+        versions of the keys.  Keys are ignored that are not `LookupKey`.
+    universe : `DimensionGraph`
```

Comment:
Type should be `DimensionUniverse` (a special subclass of `DimensionGraph` for the universe).